api/appendices/deprecated_classes.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[deprecated-classes]]
+== Deprecated Classes
+
+This page lists the classes that have been deprecated in earlier releases and have
```

Comment:
Should there be a note about how we *temporarily* support them for this release?